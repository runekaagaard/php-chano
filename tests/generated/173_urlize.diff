001- <a href="https://google.com" rel="nofollow">https://google.com</a>
001+ Fatal error: Call to undefined method DtlIter::urlize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/173_urlize.php on line 4