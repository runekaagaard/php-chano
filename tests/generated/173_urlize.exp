<a href="https://google.com" rel="nofollow">https://google.com</a>