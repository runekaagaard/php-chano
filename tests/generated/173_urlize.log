
---- EXPECTED OUTPUT
<a href="https://google.com" rel="nofollow">https://google.com</a>
---- ACTUAL OUTPUT
https://google.com
---- FAILED
