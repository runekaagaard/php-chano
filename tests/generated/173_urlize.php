<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'https://google.com')));
foreach ($items as $i) echo $i->input->urlize();
