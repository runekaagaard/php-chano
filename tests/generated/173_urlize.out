Fatal error: Call to undefined method DtlIter::urlize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/173_urlize.php on line 4